packages/devtools_app/lib/src/shared/server/_deep_links_api.dart

```diff
@@ -14,7 +14,8 @@ Future<List<String>> requestAndroidBuildVariants(String path) async {
     );
     final resp = await request(uri.toString());
     if (resp?.statusOk ?? false) {
-      return json.decode(resp!.body);
+      final jsonObject = jsonDecode(resp!.body) as List;
```

Comment:
I think this can be `return (jsonDecode(resp!.body) as List).cast<String>();`

Reply:
thanks!